.github/cla/4DCLA.md

```diff
@@ -0,0 +1,53 @@
+**⚠️ WARNING!**
```

Comment:
I do not know what happen but this .github/cla files must not be added (it will be in conflict with main branch)

Just comment the PR with the sentences, an automatic commit must be done (but maybe there is some issue because I add CLA stuff after the PR has been created)